pkg/cmd/pulumi/whoami.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/pulumi/pulumi/pkg/v3/backend"
 	"github.com/pulumi/pulumi/pkg/v3/backend/display"
+	"github.com/pulumi/pulumi/pkg/v3/version"
```

Comment:
Do we want this one or `github.com/pulumi/pulumi/sdk/v3/go/common/version`? What's the difference?

Reply:
You want this one, see what the `about` command does.